vendor.mod

```diff
@@ -0,0 +1,81 @@
+module github.com/docker/swarmkit
+
+go 1.17
+
+require (
+	code.cloudfoundry.org/clock v1.0.0
+	github.com/Microsoft/go-winio v0.4.17
+	github.com/akutz/memconn v0.1.1-0.20180504174903-e0a19f53d865
+	github.com/cloudflare/cfssl v0.0.0-20180323000720-5d63dbd981b5
+	github.com/container-storage-interface/spec v1.2.0
+	github.com/coreos/etcd v3.3.25+incompatible
+	github.com/docker/distribution v2.7.1+incompatible
+	github.com/docker/docker v20.10.3-0.20210720110456-471fd2770977+incompatible
```

Comment:
Haven't checked, so just a random comment; did you manually specify the same commits (and/or versions) for each of these, or just let `go modules` find versions? (I expect "some" will be upgraded through indirect dependencies, which we can't prevent, but as long as we're mostly at the same version, that would be great (and we can handle upgrades later where we see fit)